ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/LedgerFactory.scala

```diff
@@ -91,8 +90,7 @@ class KeyValueDeduplicationSupportFactory(
       writeServiceDelegate,
       new DeduplicationPeriodSupport(
         new CompletionBasedDeduplicationPeriodConverter(
-          domain.LedgerId(config.ledgerId),
-          completionsService,
+          completionsService
```

Comment:
Please revert this comma removal.

Reply:
Interesting, if I add it then the reformat automatically removes it. This seems to happen for single parameters calls.

Reply:
Oh you're right, because you've removed the first param. Ignore, sorry for the noise.